app/assets/src/components/ui/controls/dropdowns/bare_dropdown.scss

```diff
@@ -1,13 +1,29 @@
-:global(.ui):global(.dropdown).dropdown {
-  display: flex;
-  align-items: center;
+@import "~styles/themes/colors";
 
-  i:global(.dropdown) {
-    margin-left: 0;
+:global(.ui):global(.dropdown).bareDropdown {
```

Comment:
Does global prevent it from creating a custom name ?
Could be good to add this to dev guidelines too.